swifttool/client.py

```diff
@@ -96,26 +97,51 @@ def __init__(self, data=None):
         self.replicas = 3
         self.min_part_hours = 1
         self.zones = {}
+        # Area to build new or update existing rings
+        self.workspace = tempfile.mkdtemp()
```

Comment:
@craigtracey Cleanup still happens in bootstrap like before... What do you suggest? __del__ method?